base/file.jl

```diff
@@ -1202,6 +1214,9 @@ Specifically, currently on Windows:
 4. `rename` may remove `oldpath` if it is a hardlink to `newpath`.
 
 See also: [`mv`](@ref).
+
+!!! compat "Julia 1.12"
+    This method was publicized in Julia 1.12.
```

Comment:
```suggestion
    This method was made public in Julia 1.12.
```
I don't now if those specifically are synonyms